2022-02-10 08:30:56,035 INFO    MainThread:3890 [wandb_setup.py:_flush():75] Loading settings from /home/ubuntu/.config/wandb/settings
2022-02-10 08:30:56,035 INFO    MainThread:3890 [wandb_setup.py:_flush():75] Loading settings from /home/ubuntu/SJEOM/Brain-GBMmeta/GitHub/wandb/settings
2022-02-10 08:30:56,035 INFO    MainThread:3890 [wandb_setup.py:_flush():75] Loading settings from environment variables: {}
2022-02-10 08:30:56,035 INFO    MainThread:3890 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-02-10 08:30:56,035 INFO    MainThread:3890 [wandb_init.py:_log_setup():386] Logging user logs to /home/ubuntu/SJEOM/Brain-GBMmeta/GitHub/wandb/run-20220210_083056-1ps38mfl/logs/debug.log
2022-02-10 08:30:56,035 INFO    MainThread:3890 [wandb_init.py:_log_setup():387] Logging internal logs to /home/ubuntu/SJEOM/Brain-GBMmeta/GitHub/wandb/run-20220210_083056-1ps38mfl/logs/debug-internal.log
2022-02-10 08:30:56,035 INFO    MainThread:3890 [wandb_init.py:_jupyter_setup():336] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7ff799156750>
2022-02-10 08:30:56,035 INFO    MainThread:3890 [wandb_init.py:init():420] calling init triggers
2022-02-10 08:30:56,035 INFO    MainThread:3890 [wandb_init.py:init():427] wandb.init called with sweep_config: {}
config: {'batch_size': 32, 'epochs': 50, 'memo': 'monitorValAccuracy', 'loss_function': 'categorical_crossentropy', 'architecture': 'DenseNet121 pretrained', 'dataset': '220210_data'}
2022-02-10 08:30:56,035 INFO    MainThread:3890 [wandb_init.py:init():471] starting backend
2022-02-10 08:30:56,035 INFO    MainThread:3890 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-02-10 08:30:56,104 INFO    MainThread:3890 [backend.py:ensure_launched():219] starting backend process...
2022-02-10 08:30:56,253 INFO    MainThread:3890 [backend.py:ensure_launched():225] started backend process with pid: 26663
2022-02-10 08:30:56,254 INFO    MainThread:3890 [wandb_init.py:init():480] backend started and connected
2022-02-10 08:30:56,336 INFO    MainThread:3890 [wandb_run.py:_label_probe_notebook():921] probe notebook
2022-02-10 08:30:56,337 INFO    MainThread:3890 [wandb_run.py:_label_probe_notebook():931] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-02-10 08:30:56,337 INFO    MainThread:3890 [wandb_init.py:init():550] updated telemetry
2022-02-10 08:30:56,339 INFO    MainThread:3890 [wandb_init.py:init():581] communicating current version
2022-02-10 08:30:57,603 INFO    MainThread:3890 [wandb_init.py:init():586] got version response 
2022-02-10 08:30:57,603 INFO    MainThread:3890 [wandb_init.py:init():596] communicating run to backend with 30 second timeout
2022-02-10 08:30:57,975 INFO    MainThread:3890 [wandb_init.py:init():624] starting run threads in backend
2022-02-10 08:31:02,980 INFO    MainThread:3890 [wandb_run.py:_console_start():1827] atexit reg
2022-02-10 08:31:02,980 INFO    MainThread:3890 [wandb_run.py:_redirect():1701] redirect: SettingsConsole.WRAP
2022-02-10 08:31:02,981 INFO    MainThread:3890 [wandb_run.py:_redirect():1738] Wrapping output streams.
2022-02-10 08:31:02,981 INFO    MainThread:3890 [wandb_run.py:_redirect():1762] Redirects installed.
2022-02-10 08:31:02,981 INFO    MainThread:3890 [wandb_init.py:init():651] run started, returning control to user process
2022-02-10 08:31:08,721 INFO    MainThread:3890 [wandb_init.py:_pause_backend():299] pausing backend
2022-02-10 08:31:08,721 INFO    MainThread:3890 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-02-10 08:31:52,796 INFO    MainThread:3890 [wandb_init.py:_resume_backend():308] resuming backend
2022-02-10 08:31:52,804 INFO    MainThread:3890 [wandb_setup.py:_flush():75] Loading settings from /home/ubuntu/.config/wandb/settings
2022-02-10 08:31:52,804 INFO    MainThread:3890 [wandb_setup.py:_flush():75] Loading settings from /home/ubuntu/SJEOM/Brain-GBMmeta/GitHub/wandb/settings
2022-02-10 08:31:52,804 INFO    MainThread:3890 [wandb_setup.py:_flush():75] Loading settings from environment variables: {}
2022-02-10 08:31:52,804 INFO    MainThread:3890 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-02-10 08:31:52,804 INFO    MainThread:3890 [wandb_init.py:_log_setup():386] Logging user logs to /home/ubuntu/SJEOM/Brain-GBMmeta/GitHub/wandb/run-20220210_083152-3l1gq167/logs/debug.log
2022-02-10 08:31:52,804 INFO    MainThread:3890 [wandb_init.py:_log_setup():387] Logging internal logs to /home/ubuntu/SJEOM/Brain-GBMmeta/GitHub/wandb/run-20220210_083152-3l1gq167/logs/debug-internal.log
2022-02-10 08:31:52,804 INFO    MainThread:3890 [wandb_init.py:init():420] calling init triggers
2022-02-10 08:31:52,804 INFO    MainThread:3890 [wandb_init.py:init():427] wandb.init called with sweep_config: {}
config: {'batch_size': 32, 'epochs': 50, 'memo': 'monitorValAccuracy', 'loss_function': 'categorical_crossentropy', 'architecture': 'DenseNet121 pretrained', 'dataset': '220210_data'}
2022-02-10 08:31:52,804 INFO    MainThread:3890 [wandb_init.py:init():442] re-initializing run, found existing run on stack: 1ps38mfl
2022-02-10 08:31:52,805 INFO    MainThread:3890 [wandb_run.py:finish():1468] finishing run sjeom/Brain_GBMmeta/1ps38mfl
2022-02-10 08:31:52,806 INFO    MainThread:3890 [jupyter.py:save_history():429] not saving jupyter history
2022-02-10 08:31:52,806 INFO    MainThread:3890 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-02-10 08:31:52,806 INFO    MainThread:3890 [wandb_init.py:_jupyter_teardown():318] cleaning up jupyter logic
2022-02-10 08:31:52,806 INFO    MainThread:3890 [wandb_run.py:_atexit_cleanup():1797] got exitcode: 0
2022-02-10 08:31:52,807 INFO    MainThread:3890 [wandb_run.py:_restore():1769] restore
2022-02-10 08:31:53,242 INFO    MainThread:3890 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 782
  total_bytes: 782
}

2022-02-10 08:31:53,586 INFO    MainThread:3890 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 782
  total_bytes: 782
}

2022-02-10 08:31:54,013 INFO    MainThread:3890 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 782
  total_bytes: 7731
}

2022-02-10 08:31:54,115 INFO    MainThread:3890 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 782
  total_bytes: 7731
}

2022-02-10 08:31:54,216 INFO    MainThread:3890 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 782
  total_bytes: 7731
}

2022-02-10 08:31:54,318 INFO    MainThread:3890 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 782
  total_bytes: 7731
}

2022-02-10 08:31:54,419 INFO    MainThread:3890 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7731
  total_bytes: 7731
}

2022-02-10 08:31:54,521 INFO    MainThread:3890 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7731
  total_bytes: 7731
}

2022-02-10 08:31:54,622 INFO    MainThread:3890 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7731
  total_bytes: 7731
}

2022-02-10 08:31:54,723 INFO    MainThread:3890 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7731
  total_bytes: 7731
}

2022-02-10 08:31:54,824 INFO    MainThread:3890 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7731
  total_bytes: 7731
}

2022-02-10 08:31:55,556 INFO    MainThread:3890 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7731
  total_bytes: 7731
}

2022-02-10 08:31:56,164 INFO    MainThread:3890 [wandb_run.py:_wait_for_finish():1929] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7731
  total_bytes: 7731
}
local_info {
}

2022-02-10 08:31:57,518 INFO    MainThread:3890 [wandb_run.py:_append_files():2194] logging synced files
